packages/services/f-feature-management/test/endToEnd.test.js

```diff
@@ -11,31 +11,35 @@ const logger = {
     logInfo(){}
 }
 
-const fm = featureManagement({
-    json: JSON.stringify(config),
-    logger: logger
-});
-
-const functionMap = { bool: fm.getBooleanValue.bind(fm), int: fm.getIntegerValue.bind(fm), string: fm.getStringValue.bind(fm) };
 
 
 jest.mock('../src/trackExperiment');
 
 
 describe('End-to-End Tests', () => {
+
     describe.each(expectations.expectations)('Expectation %# - ', ({
         description, expectation, feature, valueType, context
     }) => {       
 
-        it(description, () => {
+        it(description, async () => {
+
+            const fm = await createFeatureManagementInstance({
+                json: JSON.stringify(config),
+                logger: logger
+            });
+            
+            const functionMap = { bool: fm.getBooleanValue.bind(fm), int: fm.getIntegerValue.bind(fm), string: fm.getStringValue.bind(fm) };
```

Comment:
A few readability tweaks
```suggestion
            const functionMap = {
                bool: fm.getBooleanValue.bind(fm),
                int: fm.getIntegerValue.bind(fm),
                string: fm.getStringValue.bind(fm)
            };
```